packages/vscode-extension/test/extension/commonUtils.test.ts

```diff
@@ -425,4 +425,37 @@ describe("CommonUtils", () => {
       chai.assert.isFalse(result);
     });
   });
+
+  describe("anonymizeFilePaths()", () => {
+    const sandbox = sinon.createSandbox();
+
+    afterEach(() => {
+      mockfs.restore();
+      sandbox.restore();
+    });
+
+    it("undefined", async () => {
+      const result = await commonUtils.anonymizeFilePaths();
+      chai.assert.equal(result, "");
+    });
+
+    it("happy path 1", async () => {
+      const result = await commonUtils.anonymizeFilePaths(
+        "at Object.require.extensions.<computed> [as .ts] (C:\\Users\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1621:12)"
+      );
+      chai.assert.equal(
+        result,
+        "at Object.require.extensions.<computed> [as .ts] (<REDACTED: user-file-path>:1621:12)"
```

Comment:
Can we preserve the filename like `<REDACTED: user-file-path>index.ts:1621:12`? It should be helpful to our debugging from telemetry events.